services/otel-gateway/.env

```diff
@@ -0,0 +1,3 @@
+BASIC_AUTH_USER="test"
```

Comment:
Rather than committing a .env file, which could be edited to contain secrets, it might be better to call it dotenv-sample, as we do in other places. Then add the following into the justfile and ensure it's called before other steps

```
_env:
    test -f .env || cp dotenv-sample .env
    
```